.github/workflows/gradle-wrapper-validation.yml

```diff
@@ -0,0 +1,10 @@
+name: "Validate Gradle Wrapper"
+on: [push, pull_request]
```

Comment:
not sure what the benefit of having both is here - maybe just `push`?

Reply:
Yep I agree, was just following their [example](https://github.com/gradle/wrapper-validation-action#add-a-new-dedicated-workflow) but it seems kinda pointless if it runs twice.

![image](https://user-images.githubusercontent.com/222827/74332005-c25f3100-4d8c-11ea-9225-9631e2f571c8.png)